components/infobox/wikis/counterstrike/infobox_league_custom.lua

```diff
@@ -99,6 +99,11 @@ local RESTRICTIONS = {
 		name = 'Academy Teams Only',
 		link = 'Academy Tournaments',
 		data = 'academy',
+	},
+	['national'] = {
```

Comment:
```suggestion
	national = {
```
maybe? (the same for the other keys)

Reply:
personally like it like the suggestion better (imo a bit cleaner)
but can understand your point too
the comment is completely optional :)

Reply:
don't care too much in this situation. in general, skipping [''] if non of the elements in a table needs it I think is preferred